articles/appliance/index.md

```diff
@@ -2,57 +2,57 @@
 
-###How many servers are needed?
+###Server requirements
 
-This depends on the expected traffic and availability required. In some scenarios, a single node is sufficient. The minimum highly available deployment requires three network load balanced nodes. No special load balancing logic is needed (e.g. no sticky sessions).
+The number of servers required to host the Auth0 appliance depends on the expected traffic and the desired availability. In some scenarios, a single node is sufficient. The minimum requirement for a highly available deployment is three load balanced network nodes. No special load balancing logic is needed (no sticky sessions).
 
-###What kind of network access does Auth0 need?
+###Internet access
 
-Depending on the features used in the platform, you might need to permit access to certain websites on the Internet from each of nodes. For example, if Facebook logins are needed, you will have to open connectivity from the Auth0 servers to `facebook.com`. If users are expected to login from the Internet with no VPN, then the authentication endpoints must be published.
+Depending on which features are implemented, you may need to permit access to certain websites on the internet from each node. For example, if Facebook logins are needed, you must open connectivity from the Auth0 servers to `facebook.com`. If users will be logging in from the internet (and not through a VPN), authentication endpoints must be published.
 
-The diagram below details some of these dependencies:
+The diagram below details a few of these dependencies:
 
 ![](/media/articles/appliance/overview.png)
 
-###Will Auth0 nodes deployed on-premises "call home"?
+###On-premises deployment communication with Auth0 
 
-No information is transferred outside the Auth0 nodes without consent from you. Nodes will contact external endpoints for maintenance only with your consent and supervision (e.g. to download a new version of the software, or apply an security patch).
+No information will be transferred outside the Auth0 nodes without consent from you. For maintenance purposes, nodes will contact external endpoints only with your consent and supervision (e.g. in order to download a new version of the software or apply an security patch).
 
-###Does Auth0 staff have access to the nodes?
+###Access to nodes by Auth0
 
-Not without your consent.
+The Auth0 staff does not have access to any nodes without your consent.
 
-###How often is maintenance performed?
+###Scheduled maintenance
 
-The typical update cycle is once per month. You can control the schedule of updates applied.
+The typical update cycle is once per month. You can also customize the update schedule.
```

Comment:
There is no "customization". You control the schedule, ie: you decide when these happen (when we have a call to do the actual update)